en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: news_post
+title: "Multiple vulnerabilities in RubyGems"
+author: "usa"
+translator:
+date: 2018-02-17 03:00:00 +0000
+tags: security
+lang: en
+---
+
+There are multiple vulnerabilities in RubyGems bundled by Ruby.
+It is [reported at the official blog of RubyGems](http://blog.rubygems.org/2018/02/15/2.7.6-released.html).
+
+## Details
+
+The following vulnerabilities have been reported.
+
+* Prevent path traversal when writing to a symlinked basedir outside of the root.
+* Fix possible Unsafe Object Deserialization Vulnerability in gem owner.
+* Strictly interpret octal fields in tar headers.
+* Raise a security error when there are duplicate files in a package.
+* Enforce URL validation on spec homepage attribute.
+* Mitigate XSS vulnerability in homepage attribute when displayed via gem server.
+* Prevent Path Traversal issue during gem installation.
+
+It is strongly recommended for Ruby users to take one of the following workarounds as soon as possible.
+
+## Affected Versions
+
+* Ruby 2.2 series: 2.2.9 and earlier
+* Ruby 2.3 series: 2.3.6 and earlier
+* Ruby 2.4 series: 2.4.3 and earlier
+* prior to trunk revision 62422
+
+## Workarounds
+
+If you can't upgrade Ruby itself, upgrade RubyGems to the latest version.
```

Comment:
> If you can't upgrade Ruby itself

Do we have newer Ruby version that bundles newer rubygems?

Reply:
oops